Cakes-Co/style.css

```diff
@@ -1 +1,153 @@
-/*  Add your styling here */
+* {
+  margin: 0px;
+  padding: 0px;
+}
+*::before,
+*::after {
+  box-sizing: border-box;
+  margin: 0px;
+  padding: 0px;
+}
+img {
+  width: 100%;
+  height: 100%;
+}
+:root {
+  --bar-width: 30px;
+  --bar-height: 5px;
+  --gap: 30px;
+}
+
+.logo-img {
+  width: 100px;
+  height: 100px;
+  align-self: center;
+}
+header {
+  background-color: blanchedalmond;
+  width: auto;
+  height: 140px;
+  display: flex;
+  justify-content: space-between;
+  align-items: stretch;
+  padding: 30px;
+}
+.inner-header {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: flex-end;
+}
+.motto-p {
+  text-align: right;
+  max-width: 20ch;
+}
+
+.ham-label {
+  display: flex;
+  flex-direction: column;
+  gap: 5px;
+  z-index: 3;
+}
+
+.ham-label input,
+.ham-label::before,
+.ham-label::after {
+  content: "";
+  width: var(--bar-width);
+  height: var(--bar-height);
+  background-color: black;
+}
+
+.ham-label input {
+  appearance: none;
+  outline: none;
+  pointer-events: none;
+}
+
+.menu-aside {
```

Comment:
This needs some iteration to be more like the wireframe, hamburger menu on mobile and horizontail menu on desktop